code/modules/antagonists/wizard/equipment/soulstone.dm

```diff
@@ -17,7 +17,13 @@
 
 	var/old_shard = FALSE
 	var/spent = FALSE
-	var/theme = THEME_CULT //themes include cult red, holy blue, and wizard purple
+/*
+	For themes
+	THEME_CULT = red
+	THEME_HOLY = blue
+	THEME_WIZARD = purple
+ */
+	var/theme = THEME_CULT /*** *///
```

Comment:
This isn't in the doc comment format.


Also, wtf is `/*** *///`??

Reply:
According to google it's BW in morse code.